src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.ts

```diff
@@ -199,7 +200,15 @@ export class ExamParticipationCoverComponent implements OnInit, OnDestroy {
         if (this.testRun) {
             return this.nameIsCorrect && this.confirmed && !!this.exam;
         }
-        return !!(this.nameIsCorrect && this.confirmed && this.exam && this.exam.visibleDate && this.exam.visibleDate.isBefore(this.serverDateService.now()));
+        console.log('timing test', this.serverDateService.now().add(5, 'minute').isAfter(this.exam.startDate!));
+        return !!(
+            this.nameIsCorrect &&
+            this.confirmed &&
+            this.exam &&
+            this.exam.visibleDate &&
+            this.exam.visibleDate.isBefore(this.serverDateService.now()) &&
+            this.serverDateService.now().add(5, 'minute').isAfter(this.exam.startDate!)
```

Comment:
Putting this.serverDateService.now inside a variable can remove one server call.

Reply:
I am not sure if the value would get updated every time if a have a custom variable that is only initialized once at the components initialization.

Reply:
I mean before this return statement, not for the whole class.

Reply:
I misunderstood the comment, thanks to @FuchsDominik its done :D